content/docs/ui/account-and-settings/how-to-set-up-domain-authentication.md

```diff
@@ -96,7 +96,15 @@ Any time that you send an email with a FROM address whose domain matches the dom
 
  ### 	Using automated security
 
- Automated security allows SendGrid to handle the signing of your DKIM and authentication of your SPF for your outbound email with CNAME records. This allows you to add a dedicated IP address or update your account without having to update your SPF record.
+<academy-link img="/img/SGA_SenderAuthentication750.png" courselink="https://rise.articulate.com/share/tZ8_9DYhJbWPTZN8CmsDK5GH0HPcOA-W">
+
+### Be Who You Say You Are With Sender Authentication
+
+Check out this [Academy Course]() to access actionable tips on how to build trust with mailbox providers and ensure your authentication is set up correctly. 
```

Comment:
between the () put the link to the rise course the same way you did with the image above.